application/models/elastic_artifact.py

```diff
@@ -0,0 +1,66 @@
+""" Elastic search artifact model wraps api into crud methods"""
+from ..schemas.artifact_schema import ArtifactSchema
+from .esmodel import ESModel
+
+
+class ElasticArtifact(ESModel):
+    """ Handles saving and searching """
+
```

Comment:
This is just the old Artifact. Now called ElasticArtifact to differentiate easier from NeoArtifact.